simulation/tools/sim_test_simple.cpp

```diff
@@ -236,240 +240,244 @@ void display_depth_image(const float* depth_buffer)
+    // range_likelihood_->getPointCloud(pc_out,true,camera_->pose());
     // Save in local frame
-    range_likelihood_->getPointCloud (pc_out,false,camera_->getPose ());
+    range_likelihood_->getPointCloud(pc_out, false, camera_->getPose());
     // TODO: what to do when there are more than one simulated view?
-    
+
     pcl::PCDWriter writer;
-    writer.write ("simulated_range_image.pcd", *pc_out,	false);  
+    writer.write("simulated_range_image.pcd", *pc_out, false);
     std::cout << "finished writing file\n";
-    
-//     pcl::visualization::CloudViewer viewer ("Simple Cloud Viewer");
-//     viewer.showCloud (pc_out);
 
+    //     pcl::visualization::CloudViewer viewer ("Simple Cloud Viewer");
+    //     viewer.showCloud (pc_out);
+
+    pcl::visualization::PCLVisualizer::Ptr viewer;
+    viewer = simpleVis(pc_out);
+    while (!viewer->wasStopped()) {
+      viewer->spinOnce(100);
+      std::this_thread::sleep_for(100ms);
+    }
+
+    // doesn't work:
+    //    viewer->~PCLVisualizer();
+    //    viewer.reset();
 
-  pcl::visualization::PCLVisualizer::Ptr viewer;
-  viewer = simpleVis(pc_out);
-  while (!viewer->wasStopped ())
-  {
-    viewer->spinOnce (100);
-    std::this_thread::sleep_for(100ms);
-  }    
-  
-  // doesn't work:
-//    viewer->~PCLVisualizer();
-//    viewer.reset();
-    
-    
     std::cout << "done\n";
     // Problem: vtk and opengl don't seem to play very well together
-    // vtk seems to misbehave after a little while and won't keep the window on the screen
+    // vtk seems to misbehave after a little while and won't keep the window on the
+    // screen
 
     // method1: kill with [x] - but eventually it crashes:
-    //while (!viewer.wasStopped ()){
-    //}    
-    
-    // method2: eventually starts ignoring cin and pops up on screen and closes almost 
+    // while (!viewer.wasStopped ()){
+    //}
+
+    // method2: eventually starts ignoring cin and pops up on screen and closes almost
     // immediately
     //  std::cout << "enter 1 to cont\n";
     //  cin >> pause;
     //  viewer.wasStopped ();
-    
-    // method 3: if you interact with the window with keys, the window is not closed properly
+
+    // method 3: if you interact with the window with keys, the window is not closed
+    // properly
     // TODO: use pcl methods as this time stuff is probably not cross playform
-//     struct timespec t;
-//     t.tv_sec = 100;
-//     //t.tv_nsec = (time_t)(20000000); // short sleep
-//     t.tv_nsec = (time_t)(0);  // long sleep - normal speed
-//     nanosleep (&t, NULL);
+    //     struct timespec t;
+    //     t.tv_sec = 100;
+    //     //t.tv_nsec = (time_t)(20000000); // short sleep
+    //     t.tv_nsec = (time_t)(0);  // long sleep - normal speed
+    //     nanosleep (&t, NULL);
     write_file_ = false;
   }
 }
 
 // Handle normal keys
 void
-on_keyboard (unsigned char key, int, int)
+on_keyboard(unsigned char key, int, int)
 {
   double speed = 0.1;
 
   if (key == 27)
```

Comment:
Will switch-case make better sense than if-else ladder? Or even a `std::tolower` to reduce the code density